<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EqualizerDialog</class>
 <widget class="QDialog" name="EqualizerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="gainLabel2">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="gainLabel3">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QDial" name="eqBandwidthDial3">
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Center Freq</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Bandwidth</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="eqBandwidthLabel2">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="eqBandwidthLabel1">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="eqBandwidthLabel3">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="eqCenterLabel1">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="eqCenterLabel2">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="gainLabel1">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" alignment="Qt::AlignHCenter">
    <widget class="CustomQSlider" name="eqSlider2">
     <property name="sliderPosition">
      <number>51</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignHCenter">
    <widget class="QSlider" name="eqSlider1">
     <property name="sliderPosition">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" alignment="Qt::AlignHCenter">
    <widget class="QSlider" name="eqSlider3">
     <property name="sliderPosition">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="3" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="eqCenterLabel3">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="CustomQDial" name="eqCenterDial1">
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QDial" name="eqCenterDial3">
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDial" name="eqCenterDial2">
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDial" name="eqBandwidthDial1">
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDial" name="eqBandwidthDial2">
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQDial</class>
   <extends>QDial</extends>
   <header>customqdial.h</header>
  </customwidget>
  <customwidget>
   <class>CustomQSlider</class>
   <extends>QSlider</extends>
   <header>customqslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
